packages/devtools_app/lib/src/app_size/app_size_table.dart

```diff
@@ -77,7 +77,8 @@ class _NameColumn extends TreeColumnData<TreemapNode> {
 }
 
 class _SizeColumn extends ColumnData<TreemapNode> {
-  _SizeColumn() : super('Size', alignment: ColumnAlignment.right);
+  _SizeColumn()
+      : super('Size', alignment: ColumnAlignment.right, fixedWidthPx: 100.0);
```

Comment:
nit: trailing comma in all of the modified super calls

Reply:
done